lms/djangoapps/courseware/tests/test_credit_requirements.py

```diff
@@ -122,7 +121,8 @@ def test_credit_requirements_eligible(self):
             response,
             "{}, you have met the requirements for credit in this course.".format(self.USER_FULL_NAME)
         )
-        self.assertContains(response, "Completed by {date}".format(date=self._now_formatted_date()))
+        self.assertContains(response, "Completed by {date}")
```

Comment:
This test doesn't seem to render the datetime -- I don't know enough about the test framework to know if it doesn't run the JS functions (which would be my suspicion), but it renders appropriately when tested manually/visually. This is simply a test to see that the template is rendering appropriately, less the DateUtil function actually running. So, the "Completed by {date}" field seems appropriate to me.